lib/components/viewers/trip-viewer.js

```diff
@@ -91,7 +97,7 @@ class TripViewer extends Component {
                 <SpanWithSpace margin={0.25} />
                 {tripData.bikesAllowed === 1 && (
                   <Label bsStyle="success">
```

Comment:
Would it make sense to place an `aria-label` for this entire component? (non-blocking)

Reply:
Yes that might be a better option and then have the icon be `labelledby` the parent